deeprankcore/tools/transform.py

```diff
@@ -0,0 +1,199 @@
+import h5py
+import pandas as pd
+import logging
+
+from typing import List, Union, Tuple
+from deeprankcore.domain import (
+    edgestorage as Efeat,
+    nodestorage as Nfeat,
+    targetstorage as targets)
+import numpy as np
+import matplotlib.pyplot as plt
+
+
+_log = logging.getLogger(__name__)
+
+
+def hdf5_to_pandas( # noqa: MC0001, pylint: disable=too-many-locals
+    hdf5_path: Union[str,List],
+    subset: List[str] = None,
+    node_features: Union[List[str], str] = "all",
+    edge_features: Union[List[str], str] = "all",
+    target_features: Union[List[str], str] = "all"
+) -> pd.DataFrame:
+    """
+    Args
+    ----------
+    hdf5_path (str or list): Path to hdf5 file(s). For multiple hdf5 files, 
+        insert the paths in a list.
+
+    subset (list, optional): list of keys from hdf5 file to include. Defaults to None (meaning include all).
+
+    node_features (str or list, optional): consider all pre-computed node features ("all")
+        or some defined node features (provide a list, example: ["res_type", "polarity", "bsa"]).
+        The complete list can be found in deeprankcore/domain/nodestorage.py
+
+    edge_features (list, optional): consider all pre-computed edge features ("all")
+        or some defined edge features (provide a list, example: ["dist", "coulomb"]).
+        The complete list can be found in deeprankcore/domain/edgestorage.py
+
+    target_features (list, optional): consider all pre-computed target features ("all")
+        or some defined target features (provide a list, example: ["binary", "capri_class"]).
+        The complete list (only of the pre-defined ones) can be found in deeprankcore/domain/targetstorage.py
+    Returns
+    ----------
```

Comment:
```suggestion
        The complete list (only of the pre-defined ones) can be found in deeprankcore/domain/targetstorage.py

        Returns
    ----------
```

Reply:
I inserted the line and I unified the comment to the [Google docstrings format](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html), the one we are using in the modules already on readthedocs.